packages/linear-genome-view/src/LinearGenomeView/components/OverviewRubberBand.tsx

```diff
@@ -87,14 +87,14 @@ function OverviewRubberBand({
 
     function globalMouseUp(event: MouseEvent) {
       if (controlsRef.current && startX && currentX) {
-        if (startX !== currentX)
+        if (startX !== currentX && Math.abs(currentX - startX) > 3)
```

Comment:
```suggestion
        if (Math.abs(currentX - startX) > 3)
```
this should make the left half of the conditional redundant and can be removed